modules/rhizo_base/manifests/users.pp

```diff
@@ -19,7 +19,6 @@
       gid      => 'rhizomatica',
       home     => '/home/rhizomatica',
       password => $rhizo::password_hash,
-      uid      => '1000',
```

Comment:
I'm pretty sure your correct with all the logic in the commit msg, but I want to think about why it was does like this. Maybe people before did something weird...